src/escpos/escpos.py

```diff
@@ -263,8 +263,8 @@ def image(
             )
             tone = b"0"
             colors = b"1"
-            ym = six.int2byte(1 if high_density_vertical else 2)
-            xm = six.int2byte(1 if high_density_horizontal else 2)
+            ym = b"1" if high_density_vertical else b"2"
+            xm = b"1" if high_density_horizontal else b"2"
```

Comment:
```suggestion
            xm = b"\x01" if high_density_horizontal else b"\x02"
```
Shouldn't that be hex? (Otherwise I guess you are using the byte representation of ASCII "1"

Reply:
WHOOPS !